movo_common/movo_description/urdf/movo_components/movo_pan_tilt.urdf.xacro

```diff
@@ -137,9 +137,71 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 		</link>
         
 		<xacro:sensor_d435 parent="tilt_link">
-			<origin xyz="0.06 0 -0.025" rpy="${-M_PI/2} 0 ${-M_PI/2}" />
+			<origin xyz="0.22 -0.06 -0.03" rpy="${M_PI/2} 0 ${-7*M_PI/12}" />
```

Comment:
change the first parameter of "rpy=" to ${-M_PI/2}. This change is also on another branch but it is useful here too